### PR TITLE
Pkgdown, utils, and data availability article

### PR DESCRIPTION
**Pull request by mrcaseb**

### CODE REVIEW

**Comment by mrcaseb** [comment]
Data update article doesn't cover all data sets yet, but it provides information on the mostly asked ones

File: vignettes/articles/nflverse_data_schedule.Rmd [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+)
+```
+
+`nflreadr` provides a wide range of NFL related data. This is powered by an extensive, automated data infrastructure. This article provides information on update and availability schedules, as this is a pressing question for many nflverse users, especially right before the start of a season. 
+
+## nflverse Play-by-Play Data
+
+There are three ways to obtain nflverse pbp data. 
+
+1. `nflfastR::build_nflfastR_pbp()` parses raw pbp and is usually available within 15 minutes after a game ended.
```
Comment (tanho63) [inline vignettes/articles/nflverse_data_schedule.Rmd:18]: I think I’d start in the order we’d want people to use them, eg load_pbp, then build_nflfastr_pbp(), then update_db() which is to me super niche

File: vignettes/articles/nflverse_data_schedule.Rmd [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: "nflverse Data Update and Availability Schedule"
```
Comment (tanho63) [inline vignettes/articles/nflverse_data_schedule.Rmd:2]: Maybe we could/should include the badges from nflverse/nflverse-data’s readme as well as copy this write up there as well?

File: vignettes/articles/nflverse_data_schedule.Rmd (hunk 1 above)
Comment (mrcaseb) [reply vignettes/articles/nflverse_data_schedule.Rmd:18]: I had it like this first, but both 3.) and 2.) refer to 1.) in order to explain how it works in the background

Comment (tanho63) [reply vignettes/articles/nflverse_data_schedule.Rmd:18]: I can draft a revision in a bit, am still not awake yet haha

File: vignettes/articles/nflverse_data_schedule.Rmd [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  )
+
+nflversedata::nflverse_save(data_releases, "data_releases", "nflverse Data Releases", "misc", file_types = "rds")
+```
+
+
+```{r echo = FALSE, message = FALSE, warning = FALSE}
+update_times <- nflreadr::rds_from_url("https://github.com/nflverse/nflverse-data/releases/download/misc/data_releases.rds")
+
+status <- tibble::tribble(
```
Comment (tanho63) [inline vignettes/articles/nflverse_data_schedule.Rmd:44]: I don't think this code will work without tibble installed which nflreadr doesn't have?

File: vignettes/articles/nflverse_data_schedule.Rmd [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+knitr::kable(status)
+```
+
+## nflverse Play-by-Play Data
+
+There are three ways to obtain nflverse pbp data. 
+
+1. `nflfastR::build_nflfastR_pbp()` parses raw pbp and is usually available within 15 minutes after a game ended.
+2. `nflfastR::update_db()` maintains a pbp database for the user. It runs 1.) to add new games and is therefore subject to the same update schedule.
+3. `nflreadr::load_pbp()` or `nflfastR::load_pbp()` load the complete season of data, automatically computed with 1.). The dataset updates every night (in the range of 3 am - 5 am ET) during the season.
```
Comment (tanho63) [inline vignettes/articles/nflverse_data_schedule.Rmd:77]:
```suggestion
1. We recommend using `nflreadr::load_pbp()` (and other nflreadr functions) as the primary method of retrieving data. This uses our automated data infrastructure to download clean data, and we update this on a nightly basis after each gameday during the season.  
2. If this is not quick enough, `nflfastR::build_nflfastR_pbp()` parses raw pbp json files we save, usually available within 15 minutes after a game has ended. 
3. If you have a SQL database, `nflfastR::update_db()` helps maintain a pbp database. It uses `nflfastR::build_nflfastR_pbp()` so can be run on the same schedule (i.e. within 15-30 minutes after a game has ended).  
```

**Pull request merged**